pkg/common/cns-lib/vsphere/virtualcentermanager.go

```diff
@@ -145,3 +150,36 @@ func (m *defaultVirtualCenterManager) UnregisterAllVirtualCenters(ctx context.Co
 	})
 	return err
 }
+
+func (m *defaultVirtualCenterManager) IsvSANFileServicesSupported(ctx context.Context, host string) (bool, error) {
+	log := logger.GetLogger(ctx)
+	is67u3Release, err := m.isVsan67u3Release(ctx, host)
+	if err != nil {
+		log.Errorf("Failed to identify the vCenter release with error: %+v", err)
+		return false, err
+	}
+	return !is67u3Release, nil
+}
+
+func (m *defaultVirtualCenterManager) IsExtendVolumeSupported(ctx context.Context, host string) (bool, error) {
+	log := logger.GetLogger(ctx)
+	is67u3Release, err := m.isVsan67u3Release(ctx, host)
+	if err != nil {
+		log.Errorf("Failed to identify the vCenter release with error: %+v", err)
+		return false, err
+	}
+	return !is67u3Release, nil
+}
+
+// isVsan67u3Release returns true if it is vSAN 67u3 Release of vCenter.
+func (m *defaultVirtualCenterManager) isVsan67u3Release(ctx context.Context, host string) (bool, error) {
```

Comment:
Do you want to add this to VirtualCenterManager interface?

Reply:
No, added this as a param to this function.